endpoints/openrtb2/amp_auction.go

```diff
@@ -46,6 +47,17 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 	// to compute the auction timeout.
 
 	// Set this as an AMP request in Metrics.
+
+	ao := analytics.AmpObject{
+		Type:   analytics.AMP,
+		Status: http.StatusOK,
+		Errors: make([]error, 0),
+	}
+
+	defer func() {
```

Comment:
I think you can just call `defer deps.analytics.LogAmpObject(&ao)`

Reply:
thanks, got it